tests/Resources/ti.ui.webview.test.js

```diff
@@ -0,0 +1,345 @@
+					try {
+						should(result).be.eql('WebView.evalJS.TEST');
+
+						finish();
+					} catch (err) {
+						finish(err);
+					}
+				});
+			} else { // other platforms return the result as result of function call!
+				result = webview.evalJS('Ti.API.info("Hello, World!");"WebView.evalJS.TEST";');
+				try {
+					should(result).be.eql('WebView.evalJS.TEST'); // Android reports null
+
+					finish();
+				} catch (err) {
+					finish(err);
+				}
+			}
+		});
+		win.addEventListener('focus', function () {
+			if (didFocus) {
+				return;
+			}
+			didFocus = true;
+
+			try {
+				webview.url = 'ti.ui.webview.test.html';
+			} catch (err) {
+				hadError = true;
+				finish(err);
+			}
+		});
+
+		win.add(webview);
+		win.open();
+	});
+
+	it.windows('url (ms-appx)', function (finish) {
+		var w,
+			webview;
+
+		this.timeout(10000);
+
+		w = Ti.UI.createWindow({
+			backgroundColor: 'blue'
+		});
+		webview = Ti.UI.createWebView();
+
+		webview.addEventListener('load', function () {
+			w.close();
+			finish();
+		});
+		w.addEventListener('open', function () {
+			should(function () {
+				webview.url = 'ms-appx:///ti.ui.webview.test.html';
+			}).not.throw();
+		});
+
+		w.add(webview);
+		w.open();
+	});
+
+	it.windows('url (ms-appx-web)', function (finish) {
+		var w,
+			webview;
+
+		this.timeout(10000);
+
+		w = Ti.UI.createWindow({
+			backgroundColor: 'blue'
+		});
+		webview = Ti.UI.createWebView();
+
+		webview.addEventListener('load', function () {
+			w.close();
+			finish();
+		});
+		w.addEventListener('open', function () {
+			should(function () {
+				webview.url = 'ms-appx-web:///ti.ui.webview.test.html';
+			}).not.throw();
+		});
+
+		w.add(webview);
+		w.open();
+	});
+
+	it.windows('url (ms-appx-data)', function (finish) {
+		var w,
+			webview;
+
+		this.timeout(10000);
+
+		function prepare(files) {
+			var webroot = Ti.Filesystem.applicationDataDirectory + 'webroot',
+				webroot_file = Ti.Filesystem.getFile(webroot),
+				i,
+				file,
+				from,
+				to;
+
+			if (!webroot_file.exists()) {
+				webroot_file.createDirectory();
+			}
+			for (i = 0; i < files.length; i++) {
+				file = files[i];
+				from = Ti.Filesystem.getFile(Ti.Filesystem.resourcesDirectory, file);
+				to = webroot + Ti.Filesystem.separator + file;
+				from.copy(to);
+			}
+		}
+
+		w = Ti.UI.createWindow({
+			backgroundColor: 'blue'
+		});
+		webview = Ti.UI.createWebView();
+		webview.addEventListener('load', function () {
+			w.close();
+			finish();
+		});
+		w.addEventListener('open', function () {
+			prepare([ 'ti.ui.webview.test.html' ]);
+			should(function () {
+				webview.url = 'ms-appdata:///local/webroot/ti.ui.webview.test.html';
+			}).not.throw();
+		});
+
+		w.add(webview);
+		w.open();
+	});
+
+	it('userAgent', function (finish) {
+		win = Ti.UI.createWindow({backgroundColor: 'gray'});
+		var webView = Ti.UI.createWebView({
+		        url: 'http://whatsmyuseragent.org',
```

Comment:
Indentation is with spaces for some of the next lines. Using tabs would match the rest of the file.